global/scripts/CorrThick.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+set -eu
+
+pipedirguessed=0
+if [[ ""$HCPPIPEDIR:-"" == "" ]]
+then
+	pipedirguessed=1
+	#fix this if the script is more than one level below HCPPIPEDIR
+	export HCPPIPEDIR="$(dirname -- "$0")/.."
+fi
+
+source "$HCPPIPEDIR/global/scripts/newopts.shlib" "$@"
+source "$HCPPIPEDIR/global/scripts/debug.shlib" "$@"
+
+#description of script/command
+opts_SetScriptDescription "Run curvature corrected thickness python script and save curvatures, regression coefficients, and curvature-corrected thickness"
+
+opts_AddMandatory '--subject-dir' 'SubjectDir' 'path' "folder containing all subjects"
+opts_AddMandatory '--subject' 'Subject' 'subject ID' "subject-id"
+opts_AddMandatory '--regname' 'RegName' 'my reg' "set the desired registration name(s) separated by @, 'string' 'RegName@RegName@RegName@...etc.' use MSMSulc as default"
+opts_AddOptional '--hemi' 'Hemi' 'hemisphere' "provide hemisphere for regression calculation, L=Left, R=Right, default 'L R' or B=Both" "B"
+opts_AddOptional '--surf' 'Surface' 'surface' "provide surface for regression calculation, white or midthickness, default midthickness" "midthickness"
+opts_AddOptional '--patch-size' 'PatchSize' 'patch size' "provide patch size for regression, default 6" "6"
+opts_AddOptional '--surf-smooth' 'SurfSmooth' 'surf smooth' "provide surface smoothing fwhm, default 2.14" "2.14"
+opts_AddOptional '--metric-smooth' 'MetricSmooth' 'metric smooth' "provide metric smoothing fwhm, default 2.52" "2.52"
```

Comment:
Repeating the words from the flag name as the "type" part of the help info isn't very helpful.  The regression patch is also a gaussian weighted kernel, right?
```suggestion
opts_AddOptional '--patch-size' 'PatchSize' 'distance' "provide patch kernel size in millimeters FWHM for regression, default 6" "6"
opts_AddOptional '--surf-smooth' 'SurfSmooth' 'distance' "provide surface smoothing in millimeters FWHM, default 2.14" "2.14"
opts_AddOptional '--metric-smooth' 'MetricSmooth' 'distance' "provide metric smoothing in millimeters FWHM, default 2.52" "2.52"
```